build.zig

```diff
@@ -1094,8 +1094,8 @@ fn configHeader(b: *std.Build, t: std.Target) *std.Build.Step.ConfigHeader {
         .HAVE_WCSNCASECMP = 1,
         .HAVE__WCSNICMP = 0,
         .HAVE_STRLEN = 1,
-        .HAVE_STRLCPY = 1,
-        .HAVE_STRLCAT = 1,
+        .HAVE_STRLCPY = !is_linux,
+        .HAVE_STRLCAT = !is_linux,
```

Comment:
musl provides these functions, so it'd be nice to keep using them.
```suggestion
        .HAVE_STRLCPY = !is_linux or is_musl,
        .HAVE_STRLCAT = !is_linux or is_musl,
```